lightning_pose/data/augmentations.py

```diff
@@ -97,11 +97,12 @@ def imgaug_transform(cfg: DictConfig) -> iaa.Sequential:
     else:
         raise NotImplementedError("must choose imgaug kind from 'default', 'dlc', 'dlc-top-down'")
 
-    # always apply resizing transform
-    data_transform.append(
-        iaa.Resize({
-            "height": cfg.data.image_resize_dims.height,
-            "width": cfg.data.image_resize_dims.width}
-        ))
+    # do not resize when using dynamic crop pipeline
```

Comment:
This is a little tricky to handle, because we haven't totally fleshed out the way the pipeline is going to work? But I think we are planning to have just one dataset, in which case we will want access to the full, un-resized images, although I'm not sure yet how to do this in an efficient way keeping the images on the GPU